scripts/ci/docker-compose/integration-statsd.yml

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+version: "2.2"
+services:
+  statsd-exporter:
+    image: prom/statsd-exporter
```

Comment:
We are already using airffow''s version of the statsd exporter in the helm chart and it is published in airflow dockerhub published https://github.com/apache/airflow/blob/bb00f164da6c84f078e66582217832aeedb2a966/chart/values.yaml#L136 

We also have our own way to build and publish it as needed (it is super-easy): https://github.com/apache/airflow/blob/master/chart/dockerfiles/statsd-exporter/build_and_push.sh

Shoud we use that version instead? Should we rebuild and push a new version instead?

Reply:
Sure, didn't know that!